### PR TITLE
chore: update es/pt/fr translations for asset description overrides

### PR DESCRIPTION
**Pull request by firebomb1**
It took a bit of time due to the recent changes within the Globalization WS, but here are the asset description overrides in the languages for which we currently have translators. There is a plan for additional languages to be done through bounties soon™.

As a side note, I haven't checked the logic for this library, but if there is a mechanism to fallback to English if a string is missing or empty in a language, I think it would be preferable next time (and easier on my side) if the strings in the other languages were left empty, this way translators can filter "untranslated" strings on the Gitlocalize platform. Same goes for the "web" repo strings actually.

And it might be a good thing to empty strings in other languages when there are substantial changes that you want to see reflected in other languages to a string. Due to the growing number of strings it seems unreasonable to have to review all of them manually each time they need to be updated I would think.
Would it be possible to integrate this in the engineering process when they have to update strings in English maybe ?

### CODE REVIEW

**Review by gomesalexandre — DISMISSED** [review]
Tested by linking `@shapeshiftoss/asset-service` locally, LGTM 🍬

File: packages/asset-service/src/service/descriptions/es.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
... contiene [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [3CRV](/assets/eip155:1/erc20:0x6c3f90f043a72fa612cbac8115ee6e4e90bde). [3CRV](/assets/eip155:1/erc20:0x6c3f90f043a72fa612cbac8115ee7e52bde6e490) es otro token Curve LP que consta de [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/ erc20:0xdac17f958d2ee523a2206206994597c13d831ec7), y [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48). Para adquirir crvMIM, deposite [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3), [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/erc20: 0xdac17f958d2ee523a2206206994597c13d831ec7) y/o [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48) en [Curve](https://curve.fi/mim/deposit). Tenga en cuenta que, dado que los fondos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermanentes, pero debido a que todos los activos en el fondo están vinculados a $ 1, la pérdida temporal es poco probable y solo se produciría si uno de los activos en el la pool perdió su anclaje.",
```
Comment (kaladinlight) [inline packages/asset-service/src/service/descriptions/es.json:3]:
```suggestion
  "eip155:1/erc20:0x5a6a4d54456819380173272a5e8e9b9904bdf41b": "Este token representa un grupo de liquidez en Curve, un exchange descentralizado líder. Los titulares obtienen tarifas de los usuarios que comercian en el pool y también pueden depositar sus token LP en [ShapeShift](/defi/earn) para obtener rendimiento. Este fondo de liquidez contiene [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [3CRV](/assets/eip155:1/erc20:0x6c3f90f043a72fa612cbac8115ee7e52bde6e490). [3CRV](/assets/eip155:1/erc20:0x6c3f90f043a72fa612cbac8115ee7e52bde6e490) es otro token Curve LP que consta de [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/ erc20:0xdac17f958d2ee523a2206206994597c13d831ec7), y [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48). Para adquirir crvMIM, deposite [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3), [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/erc20: 0xdac17f958d2ee523a2206206994597c13d831ec7) y/o [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48) en [Curve](https://curve.fi/mim/deposit). Tenga en cuenta que, dado que los fondos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermanentes, pero debido a que todos los activos en el fondo están vinculados a $ 1, la pérdida temporal es poco probable y solo se produciría si uno de los activos en el la pool perdió su anclaje.",
```

Comment (kaladinlight) [reply packages/asset-service/src/service/descriptions/es.json:3]: That diff looks terrible, but the `3CRV` address was different in one location

File: packages/asset-service/src/service/descriptions/es.json [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
-  "cosmos:osmosis-1/ibc:27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2": "Cosmos (ATOM) on Osmosis is a proof of stake cryptocurrency that powers an ecosystem of blockchains designed to interoperate and communicate with each other in a decentralized manner on the Osmosis blockchain, an alternative decentralized [IBC](https://tendermint.com/ibc/) blockchain."
-}
+{
+  "eip155:1/erc20:0xed4064f376cb8d68f770fb1ff088a3d0f3ff5c4d": "Este token representa un grupo de liquidez en Curve, un exchange descentralizado líder. Los titulares obtienen tarifas de los usuarios que comercian en el pool y también pueden depositar sus token LP en [ShapeShift](/defi/earn) para obtener rendimiento. Este grupo de liquidez contiene [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) y [ETH](/assets/eip155:1/slip44:60). Para adquirir Curve CRV-ETH, deposite [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) y/o [ETH](/assets/eip155:1/slip44:60) en [Curve](https:// curve.fi/crveth/deposit). Tenga en cuenta que, dado que los fondos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermanentes.",
```
Comment (kaladinlight) [inline packages/asset-service/src/service/descriptions/es.json:2]:
```suggestion
  "eip155:1/erc20:0xed4064f376cb8d68f770fb1ff088a3d0f3ff5c4d": "Este token representa un grupo de liquidez en Curve, un exchange descentralizado líder. Los titulares obtienen tarifas de los usuarios que comercian en el pool y también pueden depositar sus token LP en [ShapeShift](/defi/earn) para obtener rendimiento. Este grupo de liquidez contiene [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) y [ETH](/assets/eip155:1/slip44:60). Para adquirir Curve CRV-ETH, deposite [CRV](/assets/eip155:1/erc20:0xd533a949740bb3306d119cc777fa900ba034cd52) y/o [ETH](/assets/eip155:1/slip44:60) en [Curve](https://curve.fi/crveth/deposit). Tenga en cuenta que, dado que los fondos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermanentes.",
```

File: packages/asset-service/src/service/descriptions/es.json [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
...c9694714e2874c1ce77aa1e599461e18": "Este token representa un pool de liquidez en Curve, un exchange descentralizado líder. Los titulares obtienen tarifas de los usuarios que comercian en el pool y también pueden depositar el token LP en [ShapeShift](/defi/earn) para obtener rendimiento. Este grupo de liquidez contiene [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661). TerraUSD ([UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd)) es la moneda estable algorítmica y descentralizada de la cadena de bloques Terra. Para adquirir Curve MIM-UST, deposite [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd) en [Curve](https://curve.fi/factory/48/deposit). Tenga en cuenta que, dado que los fondos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermanente, pero debido a que todos los activos en el fondo están vinculados a $ 1, la pérdida temporal es poco probable y solo se produciría si uno de los activos en el la piscina perdió su anclaje.",
```
Comment (kaladinlight) [inline packages/asset-service/src/service/descriptions/es.json:4]:
```suggestion
  "eip155:1/erc20:0x55a8a39bc9694714e2874c1ce77aa1e599461e18": "Este token representa un pool de liquidez en Curve, un exchange descentralizado líder. Los titulares obtienen tarifas de los usuarios que comercian en el pool y también pueden depositar el token LP en [ShapeShift](/defi/earn) para obtener rendimiento. Este grupo de liquidez contiene [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd). TerraUSD ([UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd)) es la moneda estable algorítmica y descentralizada de la cadena de bloques Terra. Para adquirir Curve MIM-UST, deposite [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd) en [Curve](https://curve.fi/factory/48/deposit). Tenga en cuenta que, dado que los fondos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermanente, pero debido a que todos los activos en el fondo están vinculados a $1, la pérdida temporal es poco probable y solo se produciría si uno de los activos en el la piscina perdió su anclaje.",
```

File: packages/asset-service/src/service/descriptions/es.json [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
...8160288917a56653660e9) y [3CRV] (/assets/eIP155: 1/ERC20: 0x6c3f90f043a722612 [3CRV](/assets/eip155:1/erc20:0x6c3f90f043a72fa612cbac8115ee7e52bde6e490) es otro token Curve LP que consiste en [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/ erc20:0xdac17f958d2ee523a2206206994597c13d831ec7), y [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48). Para adquirir [crvalUSD](/assets/eip155:1/erc20:0x43b4fdfd4ff969587185cdb6f0bd875c5fc83f8c), deposite [alUSD](/assets/eip155:1/erc20:0xbc6da0fe9ad5f3b0d58160288917aa56653660e9), [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f) , [USDT](/assets/eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7), y/o [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48) en [Curve](https://curve.fi/alusd/deposit). Tenga en cuenta que, dado que los fondos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermantes, pero debido a que todos los activos en el fondo están vinculados a $ 1, la pérdida temporal es poco probable y solo se produciría si uno de los activos en el la piscina perdió su anclaje.",
```
Comment (kaladinlight) [inline packages/asset-service/src/service/descriptions/es.json:5]:
```suggestion
  "eip155:1/erc20:0x43b4fdfd4ff969587185cdb6f0bd875c5fc83f8c": "Este token representa un pool de liquidez en Curve, un exchange descentralizado líder. Los titulares obtienen tarifas de los usuarios que comercian en el pool y también pueden depositar su token LP en [ShapeShift](/defi/earn) para obtener rendimiento. Este pool de liquidez contiene [alUSD] (/assets/eip155:1/erc20:0xbc6da0fe9ad5f3b0d58160288917aa56653660e9) y [3CRV] (/assets/eip155:1/erc20:0x6c3f90f043a72fa612cbac8115ee7e52bde6e490 [3CRV](/assets/eip155:1/erc20:0x6c3f90f043a72fa612cbac8115ee7e52bde6e490) es otro token Curve LP que consiste en [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7), y [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48). Para adquirir [crvalUSD](/assets/eip155:1/erc20:0x43b4fdfd4ff969587185cdb6f0bd875c5fc83f8c), deposite [alUSD](/assets/eip155:1/erc20:0xbc6da0fe9ad5f3b0d58160288917aa56653660e9), [DAI](/assets/eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f), [USDT](/assets/eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7), y/o [USDC](/assets/eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48) en [Curve](https://curve.fi/alusd/deposit). Tenga en cuenta que, dado que los fondos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermantes, pero debido a que todos los activos en el fondo están vinculados a $1, la pérdida temporal es poco probable y solo se produciría si uno de los activos en el la piscina perdió su anclaje.",
```

**Review by kaladinlight — CHANGES_REQUESTED** [review]
After starting to review the spanish translations, there are a lot of typo/casing/spacing errors throughout that follow a very similar pattern to the ones highlighted in my first few suggested changes.

If you could review the translations again with this in mind and make the relevant updates/corrections please.

**Comment by firebomb1** [comment]
@kaladinlight Thanks a lot for the review and for having caught this!

That's my fault for being too trusting of a new contributor, even as a non-Spanish speaker I should have at least reviewed the Markdown and URLs a bit. I've done a full review of the file side-by-side with the original and corrected some other errors you did not mention, so at least the strings are consistent with each other.

File: packages/asset-service/src/service/descriptions/es.json [hunk 5]
```diff
@@ -1,23 +1,23 @@
...
+  "eip155:1/erc20:0x194ebd173f6cdace046c53eacce9b953f28411d1": "Este token representa un grupo de liquidez en Curve, un exchange descentralizado líder. Los titulares obtienen tarifas de los usuarios que comercian en el grupo y también pueden depositar su token LP en [ShapeShift](/defi/earn) para obtener rendimiento. Este grupo de liquidez contiene [EURs](/assets/eip155:1/erc20:0xdb25f211ab05b1c97d595516f45794528a807ad8) y [sEUR](/assets/eip155:1/erc20:0xd71ecff9342a5ced620049e616c5035f1db98620). Para adquirir crvEURS, deposite [EURs](/assets/eip155:1/erc20:0xdb25f211ab05b1c97d595516f45794528a807ad8) y [sEUR](/assets/eip155:1/erc20:0xd71ecff9342a5ced620049e616c5035f1db98620) en [Curve](https://curve.fi/eurs/deposit). Tenga en cuenta que, dado que los conjuntos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermanentes, pero debido a que todos los activos en el conjunto están vinculados a 1€, la pérdida impermanente es poco probable y solo se produciría si uno de los activos en el pool pierde su anclaje.",
```
Comment (kaladinlight) [inline packages/asset-service/src/service/descriptions/es.json:17]:
```suggestion
  "eip155:1/erc20:0x194ebd173f6cdace046c53eacce9b953f28411d1": "Este token representa un grupo de liquidez en Curve, un exchange descentralizado líder. Los titulares obtienen tarifas de los usuarios que comercian en el grupo y también pueden depositar su token LP en [ShapeShift](/defi/earn) para obtener rendimiento. Este grupo de liquidez contiene [EURs](/assets/eip155:1/erc20:0xdb25f211ab05b1c97d595516f45794528a807ad8) y [sEUR](/assets/eip155:1/erc20:0xd71ecff9342a5ced620049e616c5035f1db98620). Para adquirir crvEURS, deposite [EURs](/assets/eip155:1/erc20:0xdb25f211ab05b1c97d595516f45794528a807ad8) y [sEUR](/assets/eip155:1/erc20:0xd71ecff9342a5ced620049e616c5035f1db98620) en [Curve](https://curve.fi/eurs/deposit). Tenga en cuenta que, dado que los conjuntos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermanentes, pero debido a que todos los activos en el conjunto están vinculados a €1, la pérdida impermanente es poco probable y solo se produciría si uno de los activos en el pool pierde su anclaje.",
```
```suggestion
  "eip155:1/erc20:0x194ebd173f6cdace046c53eacce9b953f28411d1": "Este token representa un grupo de liquidez en Curve, un exchange descentralizado líder. Los titulares obtienen tarifas de los usuarios que comercian en el grupo y también pueden depositar su token LP en [ShapeShift](/defi/earn) para obtener rendimiento. Este grupo de liquidez contiene [EURs](/assets/eip155:1/erc20:0xdb25f211ab05b1c97d595516f45794528a807ad8) y [sEUR](/assets/eip155:1/erc20:0xd71ecff9342a5ced620049e616c5035f1db98620). Para adquirir crvEURS, deposite [EURs](/assets/eip155:1/erc20:0xdb25f211ab05b1c97d595516f45794528a807ad8) y [sEUR](/assets/eip155:1/erc20:0xd71ecff9342a5ced620049e616c5035f1db98620) en [Curve](https://curve.fi/eurs/deposit). Tenga en cuenta que, dado que los conjuntos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermanentes, pero debido a que todos los activos en el conjunto están vinculados a 1€, la pérdida impermanente es poco probable y solo se produciría si uno de los activos en el pool pierde su anclaje.",
```

File: packages/asset-service/src/service/descriptions/es.json [hunk 6]
```diff
@@ -1,23 +1,23 @@
...
+  "eip155:1/erc20:0xdc49108ce5c57bc3408c3a5e95f3d864ec386ed3": "FOXy es un Token ERC-20 de rebase, que puede intercambiar 1:1 con tokens FOX, que ofrece redistribución de rendimiento y ganancias de ShapeShift DAO. Obtenga más de sus FOX sin perder su poder de gobernanza.",
+  "cosmos:cosmoshub-4/slip44:118": "Cosmos (ATOM) es una criptomoneda que impulsa un ecosistema de cadenas de bloques diseñadas para escalar e interoperar entre sí. La visión de Cosmos se alinea con la visión de ShapeShift; un futuro de múltiples cadenas e interoperabilidad de manera descentralizada. Cosmos es una cadena de prueba de participación y permite a los titulares stake sus ATOM para ganar recompensas usando ShapeShift.",
+  "cosmos:osmosis-1/ibc:27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2": "Cosmos (ATOM) en Osmosis es una criptomoneda de prueba de participación que impulsa un ecosistema de cadenas de bloques diseñadas para interoperar y comunicarse entre sí de manera descentralizada en la Osmosis Blockchain, una cadena de bloques alternativa descentralizada [IBC] (https://tendermint.com/ibc/)."
```
Comment (kaladinlight) [inline packages/asset-service/src/service/descriptions/es.json:22]:
```suggestion
  "cosmos:osmosis-1/ibc:27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2": "Cosmos (ATOM) en Osmosis es una criptomoneda de prueba de participación que impulsa un ecosistema de cadenas de bloques diseñadas para interoperar y comunicarse entre sí de manera descentralizada en la Osmosis Blockchain, una cadena de bloques alternativa descentralizada [IBC](https://tendermint.com/ibc/)."
```
```suggestion
  "cosmos:osmosis-1/ibc:27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2": "Cosmos (ATOM) en Osmosis es una criptomoneda de prueba de participación que impulsa un ecosistema de cadenas de bloques diseñadas para interoperar y comunicarse entre sí de manera descentralizada en la Osmosis Blockchain, una cadena de bloques alternativa descentralizada [IBC] (https://tendermint.com/ibc/)."
```

**Comment by kaladinlight** [comment]
> @kaladinlight Thanks a lot for the review and for having caught this!
> 
> That's my fault for being too trusting of a new contributor, even as a non-Spanish speaker I should have at least reviewed the Markdown and URLs a bit. I've done a full review of the file side-by-side with the original to correct the errors you've mentioned, and corrected some other errors you did not mention too, so at least now the strings are consistent with each other.

No worries and thanks for looking through the rest. I found a few more small ones if you want to cross reference with the fr and pt translations

**Comment by kaladinlight** [comment]
> Thanks for these changes. I've checked Portuguese and French, they seem to be fine. The spacing and position for money units in French are actually how currencies are written usually (I've translated those like this on purpose).
> 
> I don't have such knowledge about the other two languages so I'm fine using the usual US money format as you suggested. If you prefer that we use this format in French too, just tell me I'll amend my files.

I just did a quick google search on the euro, but I am no expert haha. I biggest thing is the correct addresses obviously. I think we are good to go from here. Thanks for working through those with me!

**Review by kaladinlight — APPROVED** [review]

**Pull request merged**